RELEASE_PROCESS.md

```diff
@@ -39,15 +41,34 @@ Every commit to chef-server master is tested against a full pedant
 run. However, upgrade testing must still be done in advance of the
 release:
 
-- [ ] Run the Chef Delivery-based automated upgrade testing
-  * Checkout the `chef-server-acceptance` project from delivery.chef.co
-  * Update the `chef_server_test_url-override` attributes in
-    `.delivery/build/attributes/default.rb` to the URL corresponding to
-    the latest package you'd like to test at artifactory.chef.co
-  * Commit the change and run `delivery review`
+- [ ] Run the Chef Delivery-based automated upgrade testing.
+  * Checkout the `chef-server-acceptance` project from delivery.chef.co.
+  * Update relevant attributes in `.delivery/build/attributes/default.rb`:
+    * Update `chef_server_test_url-override` to be the URL corresponding to
+      the latest package you'd like to test at artifactory.chef.co (probably
+      a git poll build that you want to ship as the next version).
```

Comment:
"These will usually be of the form:

http://artifactory.chef.co/omnibus-current-local/com/getchef/chef-server/<version>+<timestamp>/ubuntu/14.04/chef-server-core_<version>+<timestamp>-1_amd64.deb

or something along those lines.  Remember to use the current channel instead of unstable stable on artifactory as those are the builds that passed tests on wilson."